src/stats.rs

```diff
@@ -686,9 +698,8 @@ impl Stat {
                 let inner_vals = self.group_values.read().expect("Poisoned lock");
                 let val = inner_vals.get(&tag_values);
                 if val.is_some() {
-                    val.unwrap().update(&trigger.change(defn).expect(
```

Comment:
I don't like this "unwrap". Can we get rid of it using "if let Some(mut val)" like https://gist.github.com/rust-play/f5e58ef2bed25842d4f4248d04041d87?